chainer/variable.py

```diff
@@ -434,6 +434,17 @@ def add_cand(cand):
                             x._grad += gx
             del gxs  # to reduce memory usage
 
+    def reshape(self, *shape):
+        """Returns a variable of a different shape and the same content.
+
+        .. seealso::
+           :func:`chainer.functions.reshape` for full documentation,
+
+        """
+        if len(shape) == 1 and isinstance(shape[0], tuple):
+            shape = shape[0]
```

Comment:
The condition is necessary in the case when shape is passed as a tuple.

For example, the following code would raise error without the condition.

```python
import chainer
import numpy

x = chainer.Variable(numpy.array([1,2,3] ))
x.reshape((3,))
```

By the way, this API is consistent with `numpy.ndarray.reshape`.